spec/mailers/clearance_mailer_spec.rb

```diff
@@ -29,6 +29,7 @@
     email = ClearanceMailer.change_password(user)
 
     expect(email.body.to_s).to include(link)
+    expect(email.body.to_s).to have_css('a', text: 'Change my password')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.